arwen/context/asyncCall.go

```diff
@@ -0,0 +1,286 @@
+package context
+
+import (
+	"math/big"
+
+	arwen "github.com/ElrondNetwork/arwen-wasm-vm/arwen"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/ElrondNetwork/go-ext-wasm/wasmer"
+)
+
+func (host *vmContext) handleAsyncCallBreakpoint(result wasmer.Value, err error) (*vmcommon.VMOutput, error) {
```

Comment:
Add a comment with the meaning of `err`. It seems to be the WASMER `err` returned by SC function execution?

Reply:
`err` has been removed for now, since the entire error handling needs redesign.